src/app/accessRequests/views/reviewSubServiceRequest.ejs

```diff
@@ -135,7 +135,14 @@
 
                     </fieldset>
                 </div>
-
+                <div class="govuk-warning-text">
+                    <span class="govuk-warning-text__icon" aria-hidden="true">!</span>
+                    <strong class="govuk-warning-text__text">
+                      <span class="govuk-warning-text__assistive">Warning</span>
+                      By approving this request, the access for the <%= locals.endUsersGivenName %> <%=locals.endUsersFamilyName %> will change immediatley.
```

Comment:
- remove the second **the** 
- correct spelling mistake for `immediatley` to `immediately`

Reply:
done